StackExchange/Stacks
# feat(topbar): add skip link

[STACKS-584](https://stackoverflow.atlassian.net/browse/STACKS-584)

Adds a "skip to main content" link to the topbar, enabling keyboard users to jump to the main content area.

## How to test

- Go to the [topbar page](https://deploy-preview-1692--stacks.netlify.app/product/components/topbar/) and read/review the the skip link docs
- Tab through the topbar as instructed in the docs and see the skip link appearing
- Focus away and see the link disappearing
- Ensure that the screen readers (NVDA and Voiceover) announce the skip to main content link as you navigate through the page
- Skip link if press should bring the user focus to the main content of the page

![Screenshot 2024-04-12 at 13 53 45](https://github.com/StackExchange/Stacks/assets/6231616/36bd83ab-493d-4bf8-8ffe-91e30ea9a7ae)

Next steps: release a new version of Stacks Classic and add a skip link for the Core topbar(s).
